src/NRules/NRules.Testing/RuleTestFixture.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NRules.Fluent;
+using NRules.Fluent.Dsl;
+using NRules.RuleModel;
+
+namespace NRules.Testing;
+
+public class RuleTestFixture : IRuleTestFixture
+{
+    private readonly Lazy<ISession> _lazySession;
+    private readonly IRuleActivator _ruleActivator;
+    private readonly RuleCompiler _compiler;
+    private readonly RepositorySetup _setup;
+
+    public RuleTestFixture(IRuleActivator ruleActivator, RuleCompiler compiler, IRuleAsseter asseter)
+    {
+        _ruleActivator = ruleActivator;
+        _compiler = compiler;
+        var repository = new RuleRepository
+        {
+            Activator = ruleActivator
+        };
+        _setup = new RepositorySetup(repository);
+        Verify = new RulesVerification(asseter, _setup);
+
+        _lazySession = new(CreateSession);
+
+        ISession CreateSession()
+        {
+            var ruleDefinitions = repository.GetRuleSets().SelectMany(set => set.Rules);
```

Comment:
@gritcsenko there is an extension method on IRuleRepository - GetRules

Reply:
Now I know. Thanks